content/en/docs/setup/upgrade/_index.md

```diff
@@ -1,6 +1,6 @@
 ---
 title: Upgrade
-description: Choose the upgrade guide that corresponds to the approach you previously used to install Istio.
+description: Upgrades must be done with the same approach used to install Istio. Select the upgrade guide that corresponds to the approach you previously used to install Istio.
```

Comment:
I think this might be adding to the confusion since there are 4 "approaches" under `Install`, but only 2 (istioctl or helm) under `Upgrade`. We also are still missing a way for people to move off of helm, and on to istioctl or operator. I think it might be best to leave this as is for now, and wait to get some better overall upgrade story in place.

Reply:
@ostromart WDTY?